pom.xml

```diff
@@ -66,6 +66,7 @@
     <maven.surefire.plugin.version>2.12.4</maven.surefire.plugin.version>
     <jetty.server.version>9.3.14.v20161028</jetty.server.version>
     <saxon.version>9.1.0.8</saxon.version>
+    <jgit.version>5.1.1.201809181055-r</jgit.version>
```

Comment:
This looks pretty much like a snapshot version. Is there an official release (which could be considered more stable)?